packages/grid/demo/paneProvider.stories.mdx

```diff
@@ -67,6 +67,13 @@ import { PANES } from './stories/data';
       }
     }}
     argTypes={{ defaultColumnValues: { control: false }, defaultRowValues: { control: false } }}
+    parameters={{
+      design: {
+        allowFullscreen: true,
+        type: 'figma',
+        url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=8807%3A33252'
+      }
+    }}
```

Comment:
This should be moved to the `<Meta>` block above so that it applies to both uncontrolled & controlled stories.